core/src/main/java/org/web3j/protocol/core/Ethereum.java

```diff
@@ -112,7 +112,7 @@ public interface Ethereum {
     Request<?, EthMaxPriorityFeePerGas> ethMaxPriorityFeePerGas();
 
     Request<?, EthFeeHistory> ethFeeHistory(
-            int blockCount, DefaultBlockParameter newestBlock, List<Double> rewardPercentiles);
+            String blockCount, DefaultBlockParameter newestBlock, List<Double> rewardPercentiles);
```

Comment:
I'm worried about backward compatibility, let's create a new method and marck old one as @Deprecated